gsm/gsm-makefile.patch

```diff
@@ -0,0 +1,126 @@
+diff --git a/Makefile b/Makefile
+index 2ad5e5d..c898103 100644
+--- a/Makefile
++++ b/Makefile
+@@ -30,6 +30,10 @@ WAV49	= -DWAV49
+ ######### define this, and read about the GSM_OPT_WAV49 option in the
+ ######### manual page on gsm_option(3).
+ 
++PG =
++#PG = -g -pg
++######### Profiling flags.  If you don't know what that means, leave it blank
++
+ # Choose a compiler.  The code works both with ANSI and K&R-C.
+ # Use -DNeedFunctionPrototypes to compile with, -UNeedFunctionPrototypes to
+ # compile without, function prototypes in the header files.
+@@ -44,7 +48,7 @@ WAV49	= -DWAV49
+ # CCFLAGS 	= -c -O
+ 
+ CC		= gcc -ansi -pedantic
+-CCFLAGS 	= -c -O2 -DNeedFunctionPrototypes=1 -Wall -Wno-comment
++CCFLAGS 	= -c $(RPM_OPT_FLAGS) -D_REENTRANT -DNeedFunctionPrototypes=1
```

Comment:
This seems wrong.  There will be no `RPM_OPT_FLAGS` in our environment.

Reply:
I took them from here: https://git.alpinelinux.org/aports/tree/main/gsm

Reply:
I think Ariadne's point stands. We need to understand what this patch does, and if it does nothing, we don't need it.

Reply:
The patch is necessary, it makes the build system more friendly for packaging purposes, but this needs to be documented.  I also think the `RPM_OPT_FLAGS` should be renamed to something more generic.